reference/test/matrix/csr_kernels.cpp

```diff
@@ -343,6 +343,19 @@ class Csr : public ::testing::Test {
 };
 
 
+TEST_F(Csr, CopyEmptyMatrix)
```

Comment:
Maybe move this to `dense_kernels.cpp`, as it tests code of `dense`, not `csr`

Reply:
Good point, I will do that.